lib/peeps.rb

```diff
@@ -0,0 +1,68 @@
+require 'pg'
+
+class Peeps
+
+  attr_reader :id, :time, :peep
+
+  def initialize(id:, time:, peep:)
+    @id = id
+    @time = time
+    @peep = peep
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    result = connection.exec("SELECT * FROM peeps")
+    result.map do |peep| 
+      Peeps.new(id: peep['id'], peep: peep['message'], time: peep['time'])
+    end
+  end
+
+  def self.create(peep:, time:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
```

Comment:
You're using this same piece of code in more than one place in this file. How might you get rid of this duplication?